kubectl-pg/cmd/connect.go

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright © 2019 Vineeth Pothulapati <vineethpothulapati@outlook.com>
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/remotecommand"
+	"log"
+	"os"
+	user "os/user"
+)
+
+// connectCmd represents the kubectl pg connect command
+var connectCmd = &cobra.Command{
+	Use:   "connect",
+	Short: "Connects to the shell prompt, psql prompt of postgres cluster",
+	Long:  `Connects to the shell prompt, psql prompt of postgres cluster and also to specified replica or master.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		clusterName, _ := cmd.Flags().GetString("cluster")
+		master, _ := cmd.Flags().GetBool("master")
+		replica, _ := cmd.Flags().GetString("replica")
+		psql, _ := cmd.Flags().GetBool("psql")
+		userName, _ := cmd.Flags().GetString("user")
+		dbName, _ := cmd.Flags().GetString("database")
+
+		if psql {
+			if userName == "" {
+				userInfo, err := user.Current()
+				if err != nil {
+					log.Fatal(err)
+				}
+				userName = userInfo.Username
+			}
+		}
+		if dbName == "" {
+			dbName = userName
+		}
+
+		connect(clusterName, master, replica, psql, userName, dbName)
+	},
+	Example: `
+	#connects to the master of postgres cluster
+	kubectl pg connect -c cluster -m
+
+	#connects to the random replica of postgres cluster
+	kubectl pg connect -c cluster
+
+	#connects to the provided replica number of postgres cluster
+	kubectl pg connect -c cluster -r 2
+
+	#connects to psql prompt of master for provided postgres cluster with current shell user
+	kubectl pg connect -c cluster -p
```

Comment:
```suggestion
	kubectl pg connect -c cluster -m -p
```
Now it matches the description ;)